FE-service/src/components/subpages/arena/WinnerModal.jsx

```diff
@@ -0,0 +1,46 @@
+import { createPortal } from "react-dom";
+import { Typography } from "@mui/material";
+import {
+  ModalOverlay,
+  ModalContainer,
+  ModalContent,
+  LeaveButton,
+  PokemonName,
+  PokemonImage,
+} from "./WinnerModal.styles";
+import { useTheme } from "../../../context/ThemeContext";
+import { POKEMON_IMG } from "../../../../../apiConfig";
+
+const WinnerModal = ({ winner, onClose }) => {
+  const handleClose = () => {
+    onClose();
+  };
+  const { theme } = useTheme();
+  return createPortal(
+    <ModalOverlay onClick={onClose}>
+      <ModalContainer theme={theme}>
```

Comment:
Same as above. Since we are not using "theme" at this level, useTheme should be called inside ModalContainer.